docs/sql-ref-syntax-dml-load.md

```diff
@@ -19,4 +19,37 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+The LOAD DATA statement can be used to load data from a file into a table or a partition in the table. The target table must not be temporary. A partition spec must be provided if and only if the target table is partitioned. The LOAD DATA statement is only supported for tables created using the Hive format.
```

Comment:
Back-tick the reserved words for clarity.
Nit: I'd write "..., or into a partition ..." to make sure it doesn't suggest it's the alternative to "from a file"